README.md

```diff
@@ -68,7 +68,7 @@ factory = "llm"
 labels = ["COMPLIMENT", "INSULT"]
 
 [components.llm.backend]
```

Comment:
We need to make sure the readme examples are updated correctly, many of them still contain references to `[components.llm.backend]`

Reply:
Yes, I deliberately haven't updated the readme yet as I wasn't sure if we stick with the proposed design.